assets/js/data/collections/constants.js

```diff
@@ -1 +1,2 @@
 export const STORE_KEY = 'wc/store/collections';
+export const DEFAULT_EMPTY_ARRAY = [];
```

Comment:
I see that comes from the previous PR but I didn't notice at that time. I'm curious to know what's the advantage of defining `DEFAULT_EMPTY_ARRAY` instead of directly using each time `[]`.

Reply:
Primary reason is so that any fallback returned by the selector will not trigger re-renders of props unnecessarily if used directly in components.  The danger of course, is that if consumers' mutate this array, they've effectively affected anywhere it's used.  With that in mind, should probably just remove it's usage.  Thoughts?